torchaudio/functional.py

```diff
@@ -458,68 +458,67 @@ def phase_vocoder(
     factor of ``rate``.
 
     Args:
-        complex_specgrams (Tensor): Dimension of `(..., freq, time, complex=2)`
+        complex_specgrams (Tensor): Dimension of `(..., freq, time)`
         rate (float): Speed-up factor
         phase_advance (Tensor): Expected phase advance in each bin. Dimension of (freq, 1)
 
     Returns:
-        Tensor: Complex Specgrams Stretch with dimension of `(..., freq, ceil(time/rate), complex=2)`
+        Tensor: Complex Specgrams Stretch with dimension of `(..., freq, ceil(time/rate))`
```

Comment:
based on the comment above, this would become:
```
        Tensor: Complex Specgrams Stretch, represented either as a real tensor with dimension of
            `(..., freq, ceil(time/rate), complex=2)` or a tensor of dimension `(..., freq, time)` with complex dtype.
```
thoughts?